app/views/home/_users.html.erb

```diff
@@ -5,22 +5,33 @@
 </div>
 
 <div id="filtered_user_list" class="user-list scroll-list scroll-styled">
-  <p><%= I18n.t('user.status.user_list') %></p>
-  <% @users.each do |user| %>
-    <div class="user-list-item d-flex">
-      <div class="user-avatar d-flex">
-        <span><i class="fa fa-user"></i></span>
-      </div>
-      <div class="col user-name">
-        <%= user.display_name %>
-      </div>
-      <div class="user-meeting d-flex">
-        <%= form_with model: JitsiCall.new, id: "init-call-#{user.id}" do |f| %>
-          <%= f.hidden_field :guest_id, value: user.id %>
-          <%= button_tag(type: 'submit', class: "fa fa-phone fa-flip-horizontal phone-button") do %>
+  <% if current_user.unavailable? %>
+    <div class="busy-container">
+      <p><%= I18n.t('user.status.busy_text') %></p>
+    </div>
+  <% else %>
+    <% @users.each do |user| %>
+      <% if user == @users[0] %>
+        <h4><%= I18n.t('user.status.start_talk') %></h4>
+      <% end %>
+      <% if user == @users[1] %>
+        <h4><%= I18n.t('user.status.or_maybe') %></h4>
+      <% end %>
```

Comment:
```suggestion
    <% @users.each_with_index do |user, index| %>
      <% if index == 0 %>
        <h4><%= I18n.t('user.status.start_talk') %></h4>
      <% end %>
      <% if index == 1 %>
        <h4><%= I18n.t('user.status.or_maybe') %></h4>
      <% end %>
```

After writing this I'm not so sure anymore that this is an improvement :sweat_smile:  What do you think @frcroth ?